CHANGELOG.md

```diff
@@ -165,60 +63,86 @@ testing and greater compatibility with dependent projects.
-  mechanism to help ensure compile-time validity of subscription queries. ([#584])
 - `[spec]` TLA+ for the Tendermint consensus algorithm including proof
   forks can only be caused by +1/3 Byzantine validators
   committing equivocation or amnesia attacks. ([#496])
 - `[spec]` English spec of light client attacks and evidence required to
   correctly handle them ([#526])
 - `[tendermint]` Implement `fmt::UpperHex` for `Transaction` ([#613])
-- `[testgen]` Various features and improvements to support model-based testing with 
-    the [Apalache model checker] ([#414])
+- `[tendermint/proto-compiler]` Protobuf structs generator now also accepts
+  commit IDs from the Tendermint Go repository ([#660])
+- `[testgen]` Various features and improvements to support model-based testing with
+  the [Apalache model checker] ([#414])
 
 ### IMPROVEMENTS:
 
 - [`light-client]` Start using model-based testing to test Light Client
   executions against traces emitted from the TLA+ model ([#414])
+- `[light-client]` Only require Tokio when `rpc-client` feature is enabled ([#425])
 - `[rpc]` A `WebSocketClient` is now provided to facilitate event
   subscription for a limited range of RPC events over a WebSocket connection.
   See the [Tendermint `/subscribe` endpoint's](https://docs.tendermint.com/master/rpc/#/Websocket/subscribe)
-  and the `tendermint-rpc` crate's docs for more details.
-  To access this struct you need to enable both the `client`, `subscription`
-  and `transport_websocket` features when using the `tendermint-rpc` crate.
-  ([#516])
+  and the `tendermint-rpc` crate's docs for more details ([#516])
 - `[rpc]` The subscription client interface provides a structured `Query`
   mechanism to help ensure compile-time validity of subscription queries.
   See the crate docs and [#584] for details.
 - `[rpc]` The RPC request and response types' fields are now all publicly
   accessible ([#636]).
-
-- `[tendermint | rpc | light-client]` Crates now compile to WASM on the `wasm32-unknown-unknown` and `wasm32-wasi` targets ([#463])
-- Dependency updates: 
-    - Update sled to 0.34 ([#490])
-    - Update k256 to v0.5 ([#578])
-    - Remove tai64 crate  ([#603])
+- `[rpc]` A new RPC probe (in the `rpc-probe` directory) has been added to
+  facilitate quick, pre-scripted interactions with a Tendermint node (via its
+  WebSocket endpoint). This aims to help improve testing and compatibility
+  between Tendermint in Go and Rust. ([#653])
+- `[rpc]` The `WebSocketClient` now adds support for all remaining RPC requests
+  by way of implementing the `Client` trait ([#646])
+- `[rpc]` Support for the `tx_search` RPC endpoint has been added ([#701])
+- `[rpc]` Responses that include events now automatically have their tag
+  key/value pairs decoded from base64, where previously tag key/value pairs
+  were Base64-encoded ([#717])
+- `[rpc]` Support for the `consensus_state` RPC endpoint has been added ([#719])
+- `[tendermint]` Bech32 encoding fix ([#690])
```

Comment:
```suggestion
```
(This bug+fix was within the rc releases. Between 0.16->0.17, this didn't actually change.)